### PR TITLE
Telegram Bot Integration

### PR DESCRIPTION
**Pull request by TinaMurimi**
#### What does this PR do?
Integrate Telegram Bot (Healthtools.KE) with API

#### Description of Task to be completed?
The Telegram Bot does the followinng:
1. Check to see if your doctor, nurse, or clinical officer is registered
2. Find out which facilities your NHIF card will cover in your county
3. Find the nearest doctor or health facility

#### Any background context you want to provide?
Trello card: https://trello.com/c/Yu2rd8ZI/38-htools-api-telegram-bot

#### Screenshots (if appropriate)
screen shot 2017-08-25 at 12 21 36

### CODE REVIEW

File: runtime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python-2.7.10
```
Comment (DavidLemayian) [inline runtime.txt]: Let's use `python-2.7.13`.

File: nginx.conf.template [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            proxy_set_header    X-Scheme $scheme;
+            proxy_connect_timeout       600;
+            proxy_send_timeout          600;
+            proxy_read_timeout          600;
+            send_timeout                600;
+        }
+    }
+
+    include servers/*;
+}
```
Comment (DavidLemayian) [inline nginx.conf.template]: Is this template necessary?

File: healthtools_ke_api/views/telegram_bot/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+# Note: Telegram Bot only works with HTTPS
+export BOT_WEBHOOK_URL=<https-forwarding-address-from-ngrok>
+export SERVER_IP="localhost"
+export TELEGRAM_PORT=5000
+
+# openssl req -newkey rsa:2048 -sha256 -nodes -keyout private.key -x509 -days 3650 -out cert.pem
+export CERT_FILE=</path/to/cert/file>
+export KEY_FILE=</path/to/keu/file>
+```
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/telegram_bot/README.md]: These docs should go to the README.md file at root.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
Thanks for this. The Build Query class was also I was thinking about it.

Three things:

- This should be to develop branch
- I can see health facilities changes here that I don't think are merged yet. We should not have those changes in the PR.
- Please fix any conflicts.

File: healthtools_ke_api/views/telegram_bot/README.md (hunk 3 above)
Comment (TinaMurimi) [reply healthtools_ke_api/views/telegram_bot/README.md]: Will move them there. I was avoid putting too much details on the base README.md.

File: nginx.conf.template (hunk 2 above)
Comment (TinaMurimi) [reply nginx.conf.template]: Thought it necessary in case someone else wants to work with the bot.

File: runtime.txt (hunk 1 above)
Comment (TinaMurimi) [reply runtime.txt]: Noted

**Comment by TinaMurimi** [comment]
@DavidLemayian Changes done as requested.

**Comment by DavidLemayian** [comment]
@tinamorale Looks like we have new conflicts from changes here..

**Comment by TinaMurimi** [comment]
@DavidLemayian Resolved.

**Pull request merged**